catroid/src/main/java/org/catrobat/catroid/common/LookData.java

```diff
@@ -190,16 +197,16 @@ public String getChecksum() {
 		return fileName.substring(0, 32);
 	}
 
-	protected String getPathToImageDirectory() {
+	String getPathToImageDirectory() {
 		return Utils.buildPath(Utils.buildProjectPath(ProjectManager.getInstance().getCurrentProject().getName()),
 				getSceneNameByLookData(), Constants.IMAGE_DIRECTORY);
```

Comment:
Use `Utils.buildScenePath`. It already internally encodes the project and scene names.

Eventually we should create a handful of helper methods for many scenarios. e.g. in this case `buildImagePath(Project, Scene)` and [Replace String paths by File](https://jira.catrob.at/browse/CAT-1304).